dtmi/stmicroelectronics/steval_proteus1/standard_if-1.json

```diff
@@ -0,0 +1,301 @@
+            "displayName": {
+              "en": "[Speed] Status Acceleration X:"
+            },
+            "name": "statusacc_x",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "[Speed] Status Acceleration Y:"
+            },
+            "name": "statusacc_y",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "[Speed] Status Acceleration Z:"
+            },
+            "name": "statusacc_z",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "RMS SPEED X:"
+            },
+            "name": "rmsspeed_x",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "RMS SPEED Y:"
+            },
+            "name": "rmsspeed_y",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "RMS SPEED Z:"
+            },
+            "name": "rmsspeed_z",
+            "schema": "double"
+          }
+        ]
+      }
+    },
+    {
+      "@type": "Telemetry",
+      "comment": "Frequency Domain Status X/Y/Z components [mm/s]",
+      "displayName": {
+        "en": "Frequency Domain Status  [m/sˆ2]"
+      },
+      "name": "predictivefrequencydomainstatus",
+      "schema": {
+        "@type": "Object",
+        "description": {
+          "en": "Sample format I.e.: {statusacc_x, statusacc_y, statusacc_z, freq_x, freq_y, freq_z, maxampli_x, maxampli_y, maxampli_z}."
+        },
+        "displayName": {
+          "en": "Single Predictive Frequency Domain sample."
+        },
+        "fields": [
+          {
+            "displayName": {
+              "en": "[Frequency Domain] Status Acceleration X:"
+            },
+            "name": "statusacc_x",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "[Frequency Domain] Status Acceleration Y:"
+            },
+            "name": "statusacc_y",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "[Frequency Domain] Status Acceleration Z:"
+            },
+            "name": "statusacc_z",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "Frequency Domain X:"
+            },
+            "name": "freq_x",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "Frequency Domain Y:"
+            },
+            "name": "freq_y",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "Frequency Domain Z:"
+            },
+            "name": "freq_z",
+            "schema": "double"
+          },
+          {
+            "displayName": {
+              "en": "Max Amplitude X:"
+            },
+            "name": "maxampli_x",
```

Comment:
perhaps capital A, here and below: "maxAmpli"

Reply:
[335ab63](https://github.com/Azure/iot-plugandplay-models/pull/449/commits/335ab63428b102711a254df3e3b7f64320e9601a)